src/__tests__/ParseSchema-test.js

```diff
@@ -13,15 +13,6 @@ var ParseSchema = require('../ParseSchema').default;
 var ParsePromise = require('../ParsePromise').default;
 var CoreManager = require('../CoreManager');
 
-function generateSaveMock(prefix) {
```

Comment:
Unused?

Reply:
Yeah I copied the jest test from Files when I added schema support. Forgot I left those in 😅